angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
+        '@angular-eslint/component-selector': 'error',
```

Comment:
[@angular-eslint/component-selector](http://codelyzer.com/rules/component-selector)

Reply:
This rule requires configuration that is specific to an application.

**Proposal**
Leave this rule disabled, but add it to section mentioning rules to consider.

Reply:
I agree that it doesn't make sense to enforce a specific prefix for component selectors from our shared ruleset.

It would be nice if it were possible to turn off that check but still enforce kebab-case and element selectors over attribute selectors. The codelyzer docs don't show how you'd do that but the [docs on angular-eslint GitHub](https://github.com/angular-eslint/angular-eslint/blob/ecbe68431fb73177d905676fef3df9be9c646636/packages/eslint-plugin/docs/rules/component-selector.md) say it *might* be possible. But even if we did that, apps would still have to override the entire config to provide a selector so including the rule in the shared ruleset doesn't add much benefit.

I think I've come around to agreeing with your proposal but wanted to lay out the landscape for others to comment.

Reply:
We discussed this in person. @TrevorKarjanis could you see what happens if we enforce `kebab-case` but leave `prefix` as empty? That seems preferable if it works.

Reply:
Yes, I will give it a try.

Reply:
The rule simply doesn't work without providing a prefix.

Reply:
@TrevorKarjanis this is ready to close